api/types/constants.go

```diff
@@ -458,6 +458,9 @@ const (
 
 	// AlertLink is an internal label that indicates that an alert is a link.
 	AlertLink = "teleport.internal/link"
+
+	// AlertLicenseExpired is an internal label that indicates that the license has expired.
+	AlertLicenseExpired = "teleport.internal/license-warning-expired"
```

Comment:
Something like this reads a bit better: 

```suggestion
	AlertLicenseExpired = "teleport.internal/license-expired-warning"
```